src/sinks/mod.rs

```diff
@@ -237,6 +241,10 @@ pub enum Sinks {
     #[cfg(feature = "sinks-gcp")]
     GcpPubsub(gcp::pubsub::PubsubConfig),
 
+    /// Hdfs.
```

Comment:
```suggestion
    /// HDFS.
```

**nit-picking (not blocking)**: this seems to be the proper casing for the name.